spec/datadog/appsec/waf_spec.rb

```diff
@@ -202,6 +202,41 @@
         end
       end
     end
+
+    context 'stress testing' do
```

Comment:
I would suggest to move it into a proper test file, this tests is running `Context.run` and correct file is `context_spec.rb`

Reply:
Fixed in e2717b3979ee6a13cffd51c089cd39c415bf3c91